### ISSUE TITLE
'make install' equivalent on Windows doesn't install '__init__.py' files.

### ISSUE THREAD

**Issue by biagas** [LLNL/conduit#1017]
I compiled v 0.8.4 on Windows with Visual Studio 2017. When I built the 'install' target, none of the '__init__.py' files were installed.

**Comment by cyrush** [comment]
More info from recent experience:

```
Output from the install command:
2>copying C:/A_Visit/ThirdParty/conduit-v0.8.7-build/libs/conduit/python/conduit_python_py_setup_build_install\lib\conduit\__init__.py -> C:A_Visit\ForRegression\visit-deps\windowsbuild\MSVC2017\conduit\0.8.7\python-modules\conduit
```

The backslash from C:\ was stripped off in the destination path.

This is a string escape issue. 

In this case:
`CMAKE_INSTALL_PREFIX=C:/A_Visit/ForRegression/visit-deps/windowsbuild/MSVC2017/conduit/0.8.7`

**Comment by cyrush** [comment]
resolved in #1122
